src/main/java/by/itechart/api/configuration/app/AppConfig.java

```diff
@@ -11,4 +12,13 @@
     public LocalValidatorFactoryBean validator() {
         return new LocalValidatorFactoryBean();
     }
+
+    @Bean
```

Comment:
It would be very useful to add some comment about this bean.
What are you doing here and why? It can make it easier for other developers to understand Spring's `magic` behind the scenes